docker/jenkins/run_functional_tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -xe
+
+cp docker/dockerfiles/bedrock_functional_tests Dockerfile
+docker build -t bedrock_functional_tests:$GIT_COMMIT --pull=true .
+
+docker pull selenium/hub:2.48.2
+docker pull selenium/node-firefox:2.48.2
```

Comment:
We'll want to update the tag when new releases are made. Perhaps we should use an environment variable, as we use it in four places already?